tests/test_bpc_redcap_export_mapping.py

```diff
@@ -0,0 +1,126 @@
+    with mock.patch.object(mock_syn, "get") as mock_get, mock.patch.object(
+        pd, "read_csv"
+    ):
+        bpc_export.get_mapping_data(
+            syn=mock_syn, synid_file_grs="synGRS", synid_file_dd="synDD", use_grs=True
+        )
+        mock_get.assert_called_with("synGRS")
+
+
+def test_get_mapping_data_calls_dd_if_use_grs_is_false(mock_syn):
+    with mock.patch.object(mock_syn, "get") as mock_get, mock.patch.object(
+        pd, "read_csv"
+    ):
+        bpc_export.get_mapping_data(
+            syn=mock_syn, synid_file_grs="synGRS", synid_file_dd="synDD", use_grs=False
+        )
+        mock_get.assert_called_with("synDD")
+
+
+@pytest.mark.parametrize(
+    "input_mapping, var_names, output_mapping",
+    [
+        (
+            pd.DataFrame(
+                {
+                    "Variable / Field Name": ["drugs_drug_1", "drugs_drug_2"],
+                    "Choices, Calculations, OR Slider Labels": [
+                        "D001, Aspirin | D002, Ibuprofen | D003, Paracetamol",
+                        "D004, Tylenol |",
+                    ],
+                }
+            ),
+            ["drugs_drug_1", "drugs_drug_2"],
+            {
+                "Aspirin": "D001",
+                "Ibuprofen": "D002",
+                "Paracetamol": "D003",
+                "Tylenol": "D004",
+            },
+        ),
+        (
+            pd.DataFrame(
+                {
+                    "Variable / Field Name": ["drugs_drug_1"],
+                    "Choices, Calculations, OR Slider Labels": ["D001, Aspirin | D002, Ibuprofen | D003, Paracetamol"],
+                }
+            ),
+            ["drugs_drug_1"],
+            {"Aspirin": "D001", "Ibuprofen": "D002", "Paracetamol": "D003"},
+        ),
+        (
+            pd.DataFrame(
+                {
+                    "Variable / Field Name": ["drugs_drug_1"],
+                    "Choices, Calculations, OR Slider Labels": ["D001, Aspirin|"],
+                }
+            ),
+            ["drugs_drug_1"],
+            {"Aspirin": "D001"},
+        ),
+        (
+            pd.DataFrame(
+                {
+                    "Variable / Field Name": ["ethnicity"],
+                    "Choices, Calculations, OR Slider Labels": ["1"],
+                }
+            ),
+            ["drugs_drug_1"],
+            {},
+        ),
+        (
+            pd.DataFrame(
+                {
+                    "Variable / Field Name": ["drugs_drug_1"],
+                    "Choices, Calculations, OR Slider Labels": [
+                        "D001, Aspirin (alternative) | D002, Ibuprofen | D003, Paracetamol"
+                    ],
+                    "extra_column" : ["test1", "test2"]
+                }
+            ),
+            ["drugs_drug_1"],
+            {"Aspirin": "D001", "Ibuprofen": "D002", "Paracetamol": "D003"},
+        ),
+    ],
+    ids=[
+        "multiple_drug_vars",
+        "diff_var_names",
```

Comment:
Can you explain why this test case does  diff_var_name mean here?

Reply:
Actually just removed that because we no longer need it.